rclcpp/test/rclcpp/test_intra_process_manager_with_allocators.cpp

```diff
@@ -0,0 +1,221 @@
+// Copyright 2015 Open Source Robotics Foundation, Inc.
```

Comment:
@wjwwood 
```suggestion
// Copyright 2015-2021 Open Source Robotics Foundation, Inc.
```
? I know half the file is old, but the other half is brand new.

Reply:
I basically rewrote it b4a0aa1d